infra/templates/az-hello-world/outputs.tf

```diff
@@ -1,4 +1,4 @@
 output "app_service_default_hostname" {
-  value = "http://${element(module.app_service.app_service_uri, 0)}"
+  value = "https://${element(module.app_service.app_service_uri, 0)}"
```

Comment:
Does this template use HTTPS?

Reply:
For some reason, app service output url in the portal is default https. I investigated, ran a few tests and couldn't figure out why but I made the code match what you would see if visiting your app service "overview" panel in the portal.